knife-azure.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "azure_mgmt_compute", "0.9.0"
   s.add_dependency "azure_mgmt_storage", "0.9.0"
   s.add_dependency "azure_mgmt_network", "0.9.0"
-  s.add_dependency "listen", "3.0.6"
+  s.add_dependency "listen", "3.1.5"
```

Comment:
Can we change this to `~> 3.0` rather than pin the version? Otherwise we'll have to do this again soon.